src/pages/auth/auth.css

```diff
@@ -18,11 +18,14 @@
   align-items: center;
```

Comment:
Suggestion: You could add show password icon.
![image](https://user-images.githubusercontent.com/91944618/161440139-4eb9e6d5-6df4-4c42-b0bb-6a91ff43cc5d.png)